.github/CODEOWNERS

```diff
@@ -1,11 +1,24 @@
 # See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
 
-*                                         @DataDog/apm-dotnet
-.github/workflows/profiler-pipeline.yml   @DataDog/profiling-dotnet
-/shared/                                  @DataDog/apm-dotnet @DataDog/profiling-dotnet
+# By default every team is owner
+*                                         @DataDog/apm-dotnet @DataDog/debugger-dotnet-agent @DataDog/profiling-dotnet
+
+# Tracer
 /tracer/                                  @DataDog/apm-dotnet
 /tracer/src/Datadog.Trace/Ci/             @DataDog/apm-dotnet @DataDog/ci-app-tracers
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/ @DataDog/ci-app-tracers
 # /tracer/src/Datadog.Trace/AppSec/       @DataDog/apm-dotnet @DataDog/appsec-dotnet
+
+# Profiler
 /profiler/                                @DataDog/profiling-dotnet
+.github/workflows/profiler-pipeline.yml   @DataDog/profiling-dotnet
+
+# Debugger
+/debugger                                 @DataDog/debugger-dotnet-agent
+
+# Shared code we could move to the root folder
+/tracer/build                             @DataDog/apm-dotnet @DataDog/debugger-dotnet-agent @DataDog/profiling-dotnet
 
-/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/ @DataDog/ci-app-tracers
\ No newline at end of file
+# Those do not exist yet, but I suggest we use that structure
+/docs/tracer/                             @DataDog/apm-dotnet
+/samples/tracer/                          @DataDog/apm-dotnet
```

Comment:
I'm good with this structure (same for moving tracer/build -> build), but 

1. I don't know if it's worth having specific code-owners for those (just grant to all?) 
2. If/When we do the move there's a _lot_ of places we need to update (all the markdown docs, build, public docs etc) so we could/should add code owners as part of that?

Reply:
Yes I may have been extra zealous here.
Fixed in 2e88fb140a63a15a8e3602d1308abae513de7de0